docs/reference/dapis/understand/index.md

```diff
@@ -50,47 +52,25 @@ to get the value of a dAPI.
 ## The role of Airnode
 
 Airnode is a flexible off-chain module that can support multiple protocols. Most
-noticeably are its implementation of the request-response protocol (RRP) and
-data feeds.
+noticeably is its implementation of the request-response protocol (RRP) and data
+feeds.
 
 An Airnode is owned by an API provider and integrates itself along side their
-API operations. Functionality within Airnode monitors one or more API operations
-hosted by an API provider and looks for a preset deviation of a data feed value
-(e.g., plus or minus 1%). When the deviation threshold is reached, Airnode
-pushes the new value on-chain into the beacon store. Each beacon represents a
-value from a single API operation.
+API operations. Airnode interfaces with these operations and gathers its signed
+data at the request of AirSeeker. AirSeeker uses the signed data to determine if
```

Comment:
Lowercase "s" in Airseeker (here and throughout).

Reply:
Ouch, got it